internal/project/framework/bean.go

```diff
@@ -10,21 +10,30 @@ import (
 	/**#bean*/
 	validate "demo/framework/internals/validator"
 	/*#bean.replace(validate "{{ .PkgPath }}/framework/internals/validator")**/
-
+	/**#bean*/
+	berror "demo/framework/internals/error"
+	/*#bean.replace(berror "{{ .PkgPath }}/framework/internals/error")**/
 	"github.com/go-playground/validator/v10"
 	"github.com/labstack/echo/v4"
 	"github.com/spf13/viper"
 )
 
 type Bean struct {
-	Validate        func(c echo.Context, vd *validator.Validate)
-	BeforeBootstrap func()
+	Validate                func(c echo.Context, vd *validator.Validate)
+	BeforeBootstrap         func()
+	errorHandlerMiddlewares []berror.ErrorHandlerMiddleware
 }
 
 func (b *Bean) Bootstrap() {
 	// Create a new echo instance
 	e := bootstrap.New()
 
+	errorHandlerMiddlewares := concatErrorHandlerMiddlewares(b.errorHandlerMiddlewares,
+		berror.ValidationErrorHanderMiddleware,
+		berror.APIErrorHanderMiddleware,
+		berror.HTTPErrorHanderMiddleware,
+		berror.DefaultErrorHanderMiddleware)
```

Comment:
we can also use
```
b.UseErrorHandlerMiddleware(berror.ValidationErrorHanderMiddleware)
b.UseErrorHandlerMiddleware(berror.APIErrorHanderMiddleware)
...
```
then we don't need the concat function, right?

Reply:
yes, but it make user don't need to think about framework errorHandlers, and  it make less code in `main.go`

Reply:
> we can also use
> 
> ```
> b.UseErrorHandlerMiddleware(berror.ValidationErrorHanderMiddleware)
> b.UseErrorHandlerMiddleware(berror.APIErrorHanderMiddleware)
> ...
> ```
> 
> then we don't need the concat function, right?

yes, but it make user don't need to think about framework errorHandlers, and  it make less code in `main.go`